Formula/stubby.rb

```diff
@@ -0,0 +1,84 @@
+class Stubby < Formula
+  desc "DNS privacy enabled stub resolver service based on getdns"
+  homepage "https://getdnsapi.net/blog/dns-privacy-daemon-stubby/"
+  url "https://getdnsapi.net/releases/getdns-1-1-1/getdns-1.1.1.tar.gz"
+  sha256 "fa414c30d5f2d2b2453b5cec77362b4cc0f44d440be5893233748d82bd6a1a56"
+
+  head do
+    url "https://github.com/getdnsapi/getdns.git", :branch => "develop"
+
+    depends_on "libtool" => :build
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
+  depends_on "openssl"
+  depends_on "libevent" => :recommended
+  depends_on "libuv" => :optional
+  depends_on "libev" => :optional
+
+  def install
+    if build.head?
+      system "glibtoolize", "-ci"
+      system "autoreconf", "-fiv"
+    end
+
+    args = [
+      "--with-ssl=#{Formula["openssl"].opt_prefix}",
+      "--with-trust-anchor=#{etc}/getdns-root.key",
+    ]
+    args << "--with-libevent" if build.with? "libevent"
+    args << "--with-libuv" if build.with? "libuv"
+    args << "--with-libev" if build.with? "libev"
+
+    # Current Makefile layout prevents simultaneous job execution
+    # https://github.com/getdnsapi/getdns/issues/166
+    ENV.deparallelize
+
+    system "./configure", "--prefix=#{prefix}", 
+                          "--enable-stub-only", 
+                          "--without-libidn", 
+                          "--enable-debug-daemon", 
+                          *args
+    system "make", "install"
+
+    etc.install "src/tools/stubby.conf"
+    bin.install "src/tools/stubby-setdns-macos.sh"
+  end
+
+  plist_options :startup => true, :manual => "sudo stubby -C /usr/local/etc/stubby.conf"
```

Comment:
I originally tried using #{etc} here but got an error:
```Error: undefined local variable or method `etc' for #<Class:0x007ff42ed10338>```
Can you explain why that fails and #{HOMEBREW_PREFIX} works?